web/styles/zulip.css

```diff
@@ -3316,3 +3319,15 @@ select.invite-as {
         padding: 5px;
     }
 }
+
+#unstar_all_messages,
+.emoji-popover-tab-item {
+    .zulip-icon-star {
+        position: relative;
+        top: 2px;
+    }
+}
+
+.extra-views-menu .unread_count {
+    background-color: hsl(200deg 100% 40%);
```

Comment:
So I was just about to ask about the new `--color-background-unread-counter` and whether it applies to all of the unread counts app-wide. This shows here that it doesn't. Two points there:

1. Rename the variable for its more specific context, I'm thinking.
2. Set up another variable here for this color, so we can track and apply it elsewhere as needed.